crypto/evp/evp_lib.c

```diff
@@ -14,6 +14,15 @@
 #include "internal/deprecated.h"
 
 #include <stdio.h>
+#include <string.h>
```

Comment:
Please just do this..

#include "e_os.h" /* strcasecmp */

Propagating this block of logic doesnt seem like a good idea.

Reply:
Yeah, I had copied this over from `conf_def.c` but already suspected that there should be a better way.
Changes as suggested in both files.

Reply:
At some point we may fix this mess related to strcasecmp..